src/traditional_de/automated.jl

```diff
@@ -51,28 +52,29 @@ w::Int=1, rtol=10.0, atol=2.0, τs = 1:100, metric = Euclidean()
 [^Hegger1999]: Hegger & Kantz, [Improved false nearest neighbor method to detect determinism in time series data. Physical Review E 60, 4970](https://doi.org/10.1103/PhysRevE.60.4970).
 """
 function optimal_traditional_de(s::AbstractVector, delaymethod::String= "mi_min",
-    dimensionmethod::String = "afnn"; thres::Real = 0.05, dmax::Int = 10,
-    w::Int=1, rtol=10.0, atol=2.0, τs = 1:100, metric = Euclidean())
+        dimensionmethod::String = "afnn";
+        thres::Real = 0.05, dmax::Int = 10, w::Int=1, rtol=10.0, atol=2.0, τs = 1:100,
+        metric = Euclidean(), r=2.0, w=1
+    )
 
     @assert dimensionmethod ∈ ("afnn", "fnn", "ifnn", "f1nn")
-
     τ = estimate_delay(s, delaymethod, τs)
-    γs = 1:dmax-1 # TODO: This must be updated to just dimension only
+    γs = 1:dmax-1 # TODO: This must be updated to dimension in 2.0
 
     if dimensionmethod=="afnn"
-        dimension_statistic = estimate_dimension(s, τ, γs, dimensionmethod)
+        dimension_statistic = afnn(s, τ, γs, metric)
         Y, τ = cao_embed(s, τ, dimension_statistic, thres)
         E2 = stochastic_indicator(s, τ, γs)
         flag = is_stochastic(E2, thres)
         flag && println("Stochastic signal, valid embedding NOT achieved ⨉.")
-    elseif dimensionmethod=="ifnn"
-        dimension_statistic = estimate_dimension(s, τ, γs)
-        Y, τ = fnn_embed(s, τ, dimension_statistic, thres)
     elseif dimensionmethod=="fnn"
-        dimension_statistic = estimate_dimension(s, τ, γs, dimensionmethod; rtol = rtol, atol = atol)
+        dimension_statistic = fnn(s, τ, γs; rtol, atol)
```

Comment:
We need to normalize the output!!

Reply:
I'll do that asap. Then your normalization in `test/compare_different_dimension_estimation.jl` with respect to the maximum value. It needs to get normalized `fnn`-internally with respect to all encountered distances. I'll do that asap.